kore/src/Kore/Log/ErrorEquationRightFunction.hs

```diff
@@ -76,9 +73,5 @@ instance Entry ErrorEquationRightFunction where
 instance SQL.Table ErrorEquationRightFunction
 
 -- | Error when RHS of equation is not a function pattern.
-errorEquationRightFunction ::
-    MonadLog m =>
-    Equation VariableName ->
-    m ()
-errorEquationRightFunction =
-    logError . renderText . layoutOneLine . pretty . ErrorEquationRightFunction
+errorEquationRightFunction :: Equation VariableName -> m ()
+errorEquationRightFunction = throw . ErrorEquationRightFunction
```

Comment:
Where is this caught?

Reply:
Reproducing sample error message:
```bash
$ cd test/all-path/00-basic/00-no-rules
$ make distinct-spec.k.out
$ cd path-kompiled/
$ kore-check-functions definition.kore --module PATH
$ tar -xvf kore-check-functions.tar.gz
```

`kore-check-functions.log` is empty.

`error.log`:
```
Equations
    requires:
        /* D Sfa */ \top{_}()
    argument:
        
    antiLeft:
        
    left:
        /* Fl Fn D */
        Lbl'Unds'List'Unds'{}(
            /* Fl Fn D */ Lbl'Stop'List{}(),
            /* Fl Fn D Sfa */ K:SortList{}
        )
    right:
        /* Fl Fn D Sfa */ K:SortList{}
    ensures:
        /* D Sfa */ \top{_}()
and
    requires:
        /* D Sfa */ \top{_}()
    argument:
        
    antiLeft:
        
    left:
        /* Fl Fn D Spa */
        Lbl'Unds'List'Unds'{}(
            /* Fl Fn D Spa */
            Lbl'Unds'List'Unds'{}(
                /* Fl Fn D Sfa */ K1:SortList{},
                /* Fl Fn D Sfa */ K2:SortList{}
            ),
            /* Fl Fn D Sfa */ K3:SortList{}
        )
    right:
        /* Fl Fn D Spa */
        Lbl'Unds'List'Unds'{}(
            /* Fl Fn D Sfa */ K1:SortList{},
            /* Fl Fn D Spa */
            Lbl'Unds'List'Unds'{}(
                /* Fl Fn D Sfa */ K2:SortList{},
                /* Fl Fn D Sfa */ K3:SortList{}
            )
        )
    ensures:
        /* D Sfa */ \top{_}()
match the same term.
```